### PR TITLE
Berlin-WDFT-Oct20-Katie Gracey

### PR DESCRIPTION
**Pull request by kgracey93**
All iterations complete!

### CODE REVIEW

File: src/movies.js [hunk 1]
```diff
@@ -1,15 +1,119 @@
...
+    if (array.length === 0) {
+      return 0;
+    }
+  
+    let valuesOnly = array.map(function (movies) {
+      return movies.rate;
+    });
+    console.log(valuesOnly);
+  
+    function bouncer(arr) {
```
Comment (Tyouxik) [inline src/movies.js:37]: I think you made your life harder here, a simple reduce method with an if statement inside for the edge cases would have been enough.
But it is nice that you experimented :)

File: src/movies.js [hunk 2]
```diff
@@ -1,15 +1,119 @@
...
 
 // Iteration 4: Drama movies - Get the average of Drama Movies
+function dramaMoviesRate(array) {  
+  let dramas = array.filter(function (movie) {
+    if (movie.genre.includes('Drama')) {
+      return true;
+    } 
+  });
+
+  let dramasSum = dramas.reduce(function (sum, review) {
```
Comment (Tyouxik) [inline src/movies.js:58]: You create a rateAverage function right before, you can use it here :)

**Review by Tyouxik — COMMENTED** [review]
Good job, you have a good understanding of array methods

File: src/movies.js (hunk 2 above)
Comment (kgracey93) [reply src/movies.js:58]: Thanks for the feedback I really appreciate it!

**Pull request closed without merging**